compose-builder/add-security.yml

```diff
@@ -196,7 +215,7 @@ services:
       - kong-migrations
 
   edgex-proxy:
-    image: ${CORE_EDGEX_REPOSITORY}/docker-edgex-security-proxy-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    image: ${CORE_EDGEX_REPOSITORY}/docker-security-security-proxy-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
```

Comment:
Wrong name. Extra `security`.  Change to `docker-security-proxy-setup-go`

Reply:
Done